enhancements/centralized-manifest-openapi-generation.md

```diff
@@ -0,0 +1,100 @@
+---
+title: centralized-manifest-openapi-generation
+authors:
+  - "@damemi"
+reviewers:
+  - "@deads2k"
+  - "@sttts"
+approvers:
+  - "@deads2k"
+  - "@sttts"
+creation-date: 2019-09-23
+last-updated: 2019-09-23
+status: provisional
+---
+
+# Centralized CRD Manifest OpenAPI Generation
+
+## Release Signoff Checklist
+- [ ] Enhancement has been determined `implementable`
+- [ ] Design details are appropriately documented
+- [ ] Test plan is defined
+
+## Summary
+This document proposes migrating operator CRD manifests (and, more importantly, the work associated with updating them)
+to be colocated with the Golang types that define them in the [openshift/api](https://github.com/openshift/api) repository.
+
+## Motivation
+We have many operators that define custom resources, but almost all of those resources are based on Golang types defined in
```

Comment:
not just operators.  The openshift/api is made up of many custom resources.

Reply:
yeah I wasn't implying that openshift/api is only operator resources, but that all the operator resources are in openshift/api. I'll clarify the wording